_blog/index.html

```diff
@@ -0,0 +1,15 @@
+---
+layout: compress
+---
+{% include home.html %}
+
+<!DOCTYPE html>
+<html lang="en-US">
+  <meta charset="utf-8">
+  <title>Redirecting…</title>
+  <link rel="canonical" href="{{home}}/blog/posts/{{site.data.istio.latest_blog_post}}">
+  <meta http-equiv="refresh" content="0; url={{home}}/blog/posts/{{site.data.istio.latest_blog_post}}">
```

Comment:
that’s still a hand crafted redirec

Reply:
Yes, if you go to istio.io/blog, you'll get a redirect to the latest post. But if you click on the link in the nav bar, then you no longer get a redirect.